src/main/java/seedu/address/security/SecurityManager.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.security;
+
+import seedu.address.commons.core.ComponentManager;
+import seedu.address.commons.events.security.SuccessfulLoginEvent;
+import seedu.address.commons.events.security.UnsuccessfulLoginEvent;
+
+/***
+ *  Implements a Security authentication that identifies user
+ */
+
+public class SecurityManager extends ComponentManager implements Security {
+    private boolean isAuthenticated;
+    private String username;
+    private String password;
+    //Person userPerson;
+
+
+    public SecurityManager(boolean isTest) {
```

Comment:
It seems kind of problematic that you have a public constructor that essentially lets you bypass all security by just calling `SecurityManager(true)`